doc/howto/customlog.md

```diff
@@ -0,0 +1,32 @@
+[&lt; EON Documentation index](/doc/index.md) 
+# EON logging configuration
+
+## Simple logging customization
+
+Edit the following variables in the node configuration file to customize path and level of logging:
+
+- **sparkz.logDir**
+
+  Path of the logging directory
+
+- **sparkz.logInfo.logFileName**
+
+  Name of the log file
+
+- **sparkz.logInfo.logFileLevel** and **sparkz.logInfo.logConsoleLevel**
+
+  Level of logging used for file and console appenders. 
+  Can be one of the following: off, fatal, error, warn, info, debug, trace, all
```

Comment:
@paolocappelletti we can also mention that his can be adjusted via ENV vars under .env file. Here is the template from config file. We can probably use ENV var for path and file name as well so you do not need to modify config file within the container

```
logInfo {
       logFileName = "debug.log"
       # levels: off, fatal, error, warn, info, debug, trace, all
       logFileLevel = "$SCNODE_LOG_FILE_LEVEL"
       logConsoleLevel = "$SCNODE_LOG_CONSOLE_LEVEL"
    }
```

Reply:
done